test/String.cpp

```diff
@@ -9,6 +9,14 @@ TEST_CASE("RCString::Copy", "[string]") {
         rcStr.Copy(str);
         REQUIRE(!SStrCmp(str, rcStr.GetString(), STORM_MAX_STR));
     }
+
+    SECTION("copies source RCString") {
+        RCString rcStr1;
+        rcStr1.Copy("foo");
+        RCString rcStr2;
+        rcStr2 = rcStr1;
```

Comment:
This isn't testing `Copy(const RCString& source)`. I think we probably want this instead:
```suggestion
        RCString rcStr1;
        rcStr1.Copy("foo");
        RCString rcStr2;
        rcStr2.Copy(rcStr1);
```